src/mixer/basetrackplayer.cpp

```diff
@@ -376,6 +432,30 @@ TrackPointer BaseTrackPlayerImpl::getLoadedTrack() const {
     return m_pLoadedTrack;
 }
 
+void BaseTrackPlayerImpl::slotCloneDeck(const QString& group) {
+    EngineChannel* pChannel = m_pEngineMaster->getChannel(group);
+    if (!pChannel) {
+        return;
+    }
+
+    slotCloneChannel(pChannel);
+}
+
+void BaseTrackPlayerImpl::slotCloneChannel(EngineChannel* pChannel) {
+    m_cloneFromChannel = pChannel;
+    if (!m_cloneFromChannel) {
+        return;
+    }
+
+    TrackPointer pTrack = m_cloneFromChannel->getEngineBuffer()->getLoadedTrack();
```

Comment:
The `m_cloneFromChannel` name reads more like a function name than a variable name to me. How about renaming this variable to `m_channelToCloneFrom`?